addons/l10n_pl_extended/models/account_move.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models, fields, api
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    taxable_supply_date = fields.Date(default=fields.Date.today())
+
+    @api.depends('taxable_supply_date')
+    def _compute_date(self):
+        super()._compute_date()
+        for move in self:
+            if move.country_code == 'PL' and move.taxable_supply_date and move.state == 'draft' and not move.statement_line_id:
```

Comment:
this make no sense too, because why you would check `move.taxable_supply_date` - you have a default on a field?

it is quite difficult to remove its value... 

then, if you change date or invoice_date - you are risking very serious errors when taxable_supply_date having old value - I've seen it so many times

Reply:
Don’t waste too much time on this. We had a round of tests yesterday and already did the observations. We can invite you to test again when we have reached a point that we consider functionally valid. 😉

Reply:
Dear Krystian, we are already considering this separately. One topic, one Pull Request. But I can confirm, this is already an issue we are considering.